{
    "type": "bundle",
    "id": "bundle--913c1a1c-49ce-4563-86f7-64cdfb822b5b",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--94866d6f-b8cb-44f4-b7bd-5d22a5f48cda",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-12-30T00:22:03.616038Z",
            "modified": "2024-12-30T00:22:03.616038Z",
            "name": "CVE-2024-56606",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\naf_packet: avoid erroring out after sock_init_data() in packet_create()\n\nAfter sock_init_data() the allocated sk object is attached to the provided\nsock object. On error, packet_create() frees the sk object leaving the\ndangling pointer in the sock object on return. Some other code may try\nto use this pointer and cause use-after-free.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-56606"
                }
            ]
        }
    ]
}